iam-login-service/src/main/webapp/WEB-INF/views/iam/login.jsp

```diff
@@ -72,62 +72,129 @@
             <h3>Welcome to <strong>${iamOrganisationName}</strong>
       </h3>
         </div>
+
+        <c:if
+            test="${
+                (
+                    loginPageConfiguration.localAuthenticationVisible
+                    or param.sll != null
+                ) && loginPageConfiguration.defaultLoginPageLayout
+            }"
+        >
+            <jsp:include page="login-form.jsp" />
+        </c:if>
         
-        <c:if test="${loginPageConfiguration.localAuthenticationVisible or param.sll != null}">
-          <jsp:include page="login-form.jsp" />
+        <c:if
+            test="${
+                !loginPageConfiguration.defaultLoginPageLayout
```

Comment:
Isn't this equivalent to `!loginPageConfiguration.defaultLoginPageLayout || !loginPageConfiguration.localAuthenticationVisible && param.ssl == null`?

Reply:
Simplified the logic. Done